PLUGINS.md

```diff
@@ -7,91 +7,120 @@ This plugin gathers BlazingMQ broker statistic and sends it to [Prometheus](http
 
 ### Configuration
 By default, plugin is disabled. To enable and configure it, edit `bmqbrkcfg.json` file as follows:
+
 1. Enable plugin and provide path to plugin library
-```json
-"appConfig": {
-    ...
-
-    "plugins": {
-        "libraries": ["<path-to-prometheus-plugin-library-folder>"],
-        "enabled": ["PrometheusStatConsumer"]
-    }
-}
-```
+   ```json
+   "appConfig": {
+       ...
+       "plugins": {
+           "libraries": ["<path-to-prometheus-plugin-library-folder>"],
+           "enabled": ["PrometheusStatConsumer"]
+       }
+   }
+   ```
+
 2. Provide plugin configuration
-```json
-"appConfig": {
-    "stats": {
-        "plugins": [
-            ...
-            {
-                "name": "PrometheusStatConsumer",
-                "publishInterval": 10,
-                "prometheusSpecific": {
-                    "host": "localhost",
-                    "port": 9091,
-                    "mode": "E_PUSH"    
-                }
-            }
-        ],
-        ...
-    }
-}
-```
+   ```json
+   "appConfig": {
+       "stats": {
+           "snapshotInterval": 1,
+           "appIdPostingDomains": ["fanout-domain-1-name", "fanout-domain-2-name"],
```

Comment:
What is the purpose of `"fanout-domain-1-name", "fanout-domain-2-name"`?

Reply:
That's only for instance. The `queue.confirm_time_max` and `queue.queue_time_max` metrics will have `AppId` tag only for queues in these domains